webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx

```diff
@@ -64,17 +115,43 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
       }
 
       out.push({
+        onPointClick,
+        plugins: [
+          closestPointPlugin({
+            checkpointsDict,
+            diamond: true,
+            onPointClick,
+            yScale: 'y',
+          }),
+          drawPointsPlugin(checkpointsDict),
```

Comment:
Are we going to want to have drawPoints only on the TrialDetailsOverview? Maybe it would make sense to always have this plugin, have checkpoints be an empty array by default, and then we don't need to pass in plugins from the page like this?

Reply:
If we wanted to do checkpoint indicators for multi-trial, we would need to do a different structure, since we are using `batches` as the index here, and would need to use `trial_id, batches` as the index there. 

I think it makes the most sense to handle this different business logic with plugins, rather than handling all the different business logic variations within the `ChartGrid` component.

We can still ensure visual consistency across charts by consolidating plugins and their usage.